### PR TITLE
Fix CAPP Agile Learning content to make sure relative links work

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Having two ways of implementing the content of the CAPP Agile Learning pages resulted in troubles with the search and translations. 
This was fixed by moving search to the top of the config file in #1024, but this gave troubles with relative links:

```
- ./build/media/index.html
  *  internally linking to pdf/2016-Nederlandse-kennis-en-innovatie-naar-de-Duitse-Zorgmarkt.pdf, which does not exist (line 24)
```

To fix it I implement the content of the CAPP Agile Learning pages for every locale in the same way. No more errors.

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
I'll fix hound issues.

File: config.rb [hunk 1]
```diff
@@ -205,6 +154,57 @@
 
 activate :directory_indexes
 
+# Search
+activate :search do |search|
```
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply config.rb]: Not much I can do bout that I think

File: config.rb [hunk 2]
```diff
@@ -205,6 +154,63 @@
...
+  end
+end
+
+def to_store(resource)
+  if resource.url.start_with?("/blog/")
+    return "Blog"
+  elsif resource.url.start_with?("/jobs/")
+    return t("search.vacancy")
+  else
+    return t("search.page")
```
[the remark this answers is not in the document]
Comment (fatboypunk) [reply config.rb]: Ah so they're not needed, sorry!

File: config.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (fatboypunk) [reply config.rb]: it's fine for now,

File: config.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply config.rb]: Thanks!

**Comment by RonaldDijkstra** [comment]
So, ready!

**Pull request merged**